github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits.yaml

```diff
@@ -2,6 +2,7 @@
 presubmits:
   k8snetworkplumbingwg/ovs-cni:
     - name: pull-e2e-ovs-cni
+      context: pull-e2e-ovs-cni
```

Comment:
This is not required since it's the default. Also people might think that they need to adjust the context also, no? Maybe you can add it to the file for the older release branch instead, since I think it's missing there.

Reply:
https://github.com/kubernetes/test-infra/blob/f6c02074bb4274351972b7313f64de61ab7bcd87/prow/config/jobs.go#L323

Reply:
I see your point. I added a little comment to that line and added context to 0.19 too

Reply:
@dhiller I can remove it completely if you mind the context on main, your call

Reply:
I think we are good. In your case using the configforker sounds like overkill.